os/migrating-to-clcs.md

```diff
@@ -0,0 +1,373 @@
+```
+
+The flannel section in a CLC is used the same way, and a version can optionally be specified for flannel as well.
+
+```container-linux-config
+flannel:
+  version:     0.7.0
+  etcd_prefix: "/coreos.com/network2"
+```
+
+### locksmith
+
+The `coreos.locksmith.*` section in a cloud config can be used to configure the locksmith daemon via environment variables.
+
+```cloud-config
+#cloud-config
+
+coreos:
+  locksmith:
+      endpoint: "http://example.com:2379"
+```
+
+Locksmith can be configured in the same way under the locksmith section of a CLC, but some of the accepted options are slightly different. Also the reboot strategy is set in the locksmith section, instead of the update section. Check out the [CLC schema][ct-config] to see what options are available.
+
+```container-linux-config
+locksmith:
+  reboot_strategy: "reboot"
+  etcd_endpoints:  "http://example.com:2379"
+```
+
+### update
+
+The `coreos.update.*` section can be used to configure the reboot strategy, update group, and update server in a cloud config.
+
+```cloud-config
+#cloud-config
+coreos:
+  update:
+    reboot-strategy: "etcd-lock"
+    group:           "stable"
+    server:          "https://public.update.core-os.net/v1/update/"
+```
+
+In the update section in a CLC the group and server can be configured, but the reboot-strategy option has been moved under the locksmith section.
+
+```container-linux-config
+update:
+  group:  "stable"
+  server: "https://public.update.core-os.net/v1/update/"
+```
+
+### units
+
+The `coreos.units.*` section in a cloud config can define arbitrary systemd units that should be started after booting.
+
+```cloud-config
+#cloud-config
+
+coreos:
+  units:
+    - name: "docker-redis.service"
+      command: "start"
+      content: |
+        [Unit]
+        Description=Redis container
+        Author=Me
+        After=docker.service
+
+        [Service]
+        Restart=always
+        ExecStart=/usr/bin/docker start -a redis_server
+        ExecStop=/usr/bin/docker stop -t 2 redis_server
+```
+
+This section could also be used to define systemd drop-in files for existing units.
+
+```cloud-config
+#cloud-config
+
+coreos:
+  units:
+    - name: "docker.service"
+      drop-ins:
+        - name: "50-insecure-registry.conf"
+          content: |
+            [Service]
+            Environment=DOCKER_OPTS='--insecure-registry="10.0.1.0/24"'
+```
+
+And existing units could also be started without any further configuration.
+
+```cloud-config
+#cloud-config
+
+coreos:
+  units:
+    - name: "etcd2.service"
+      command: "start"
+```
+
+One big difference in CLCs compared to cloud configs is that the configuration is applied via [Ignition][ignition] before the machine has fully booted, as opposed to cloudinit that runs after the machine has fully booted. As a result units cannot be directly started in a CLC, the unit is instead enabled so that systemd will begin the unit once systemd starts.
```

Comment:
This example doesn't have the `[Install]` section. We should probably call this out as well.